src/features/canvassAssignments/components/PlaceDialog/pages/Place.tsx

```diff
@@ -33,10 +39,16 @@ const Place: FC<PlaceProps> = ({
   const [adding, setAdding] = useState(false);
   const { addHousehold } = usePlaceMutations(orgId, place.id);
 
+  const numVisitedHouseholds =
+    place?.households.filter((household) =>
+      household.visits.some((visit) => visit.canvassAssId == assignment.id)
+    ).length ?? 0;
+
   return (
     <PageBase
       onClose={onClose}
       onEdit={onEdit}
+      subtitle={`${numVisitedHouseholds} / ${place.households.length} households visited`}
```

Comment:
Adding the `subtitle `is a great touch, it really enhances the clarity of the household "state" and makes it much easier to understand! 💯

Reply:
Agreed! 💯